optimization/optimization_pr.md

```diff
@@ -0,0 +1,399 @@
+
+Running a second optimization results a better fit in the region characterized by the large negative trough, and overall RMSE drops to 14.07. However, several parameters did not change as much as in the first optimization. One that continued to increase by over 200% was the layer 2/3 pyramidal NMDA weight for the distal 1 input.
+
+<div style="background-color:rgba(0, 0, 0, 0); margin-top:20px; text-align: center; vertical-align: middle; margin-bottom:40px;">
+
+<h3>Figure 8</h3>
+
+<a href="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image10.png">
+<img src="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image10.png" alt="image10" width=80%/>
+</a>
+<p>A second optimization round reducing RMSE from 17.65 to 14.07
+</p>
+</div>
+
+<div style="background-color:rgba(0, 0, 0, 0); text-align:center; vertical-align: middle; margin-top:20px; margin-bottom:40px">
+
+<table style="border-collapse: collapse; border: none;">
+
+<tr style="border: none;">
+<h3>Figure 9</h3>
+</tr>
+
+<tr style="border: none;">
+
+<td style="border: none;" width="33%">
+<a href="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image11.png"><img src="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image11.png" alt="image11" width=100%/>
+</a>
+<p>  </p>
+</td>
+
+<td style="border: none;" width="33%">
+<a href="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image12.png"><img src="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image12.png" alt="image12" width=100%/>
+</a>
+<p>  </p>
+</td>
+
+<td style="border: none;" width="33%">
+<a href="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image13.png"><img src="https://raw.githubusercontent.com/jonescompneurolab/hnn-tutorials/master/optimization/images/image13.png" alt="image13" width=100%/>
+</a>
+<p>  </p>
+</td>
+
+</tr>
+
+</table>
+Parameter values and relative changes in the optimization shown in Figure 8.
+</div>
+
+We performed a third optimization round with the resultant fit shown in Figure 10. Finally, the simulation output matches the negative trough in the experimental data well and RMSE is reduced to 10.40. We can see that the layer 2/3 pyramidal NMDA weight for the distal 1 input increased again, by 174% this time. At this point, it has changed over 2500% from its baseline value!
```

Comment:
really nice improvement in reduced RMSE after several rounds of optimization. The 2500% change in NMDA brings to mind whether initial values were appropriate and/or how to guide users towards physiologically plausible parameter values...